### PR TITLE
Add Guide to retrieve controllers addresses + their permissions

### PR DESCRIPTION
**Pull request by CJ42**
A lot of developers from the community ask how to retrieve the list of addresses that have some permissions set for the Universal Profile.

- [x] created + add this guide under **Guides > Key Manager** to show how to do this using _erc725.js_
- [x] edit LSP6 Address Permissions array diagram error (array length is `4` ✅ , not `5` ❌ 
- edited title of guide _"Execute Relay Call"_ to > _"Execute Relay Transactions"_

### CODE REVIEW

File: docs/guides/key-manager/get-controllers.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+## Setup
+
+```bash
+npm install erc725.js
+```
+
+## Step 1 - setup web3.js and erc725.js
+
+The first step is to setup both and _web3.js_ and _erc725.js_, connected to the LUKSO L14 network. We will also need the address of the Universal Profile that we want to get the controller addresses from.
```
Comment (fhildeb) [inline docs/guides/key-manager/get-controllers.md:25]:
```suggestion
The first step is to set up both _web3.js_ and _erc725.js_, and connect to the LUKSO L14 network. We will also need the address of the Universal Profile that we want to get the controller addresses from.
```

File: docs/guides/key-manager/get-controllers.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+const erc725 = new ERC725(
+  LSP6Schema,
+  myUniversalProfileAddress,
+  web3.currentProvider,
+);
+```
+
+## Step 2 - Get the list of controller addresses
+
+The next step is to retrieve the list of controller addresses. In other words, addresses that have some permissions set and are allowed to interact with the Universal Profile.
```
Comment (fhildeb) [inline docs/guides/key-manager/get-controllers.md:47]:
```suggestion
The next step is to retrieve the list of controller addresses. In other words, addresses with some permissions can interact with the Universal Profile.
```

File: docs/guides/key-manager/get-controllers.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+All the controller addresses are listed under the data key `AddressPermissions[]`. We can therefore obtain this list of addresses by:
+
+1. querying the `AddressPermissions[]` to get the array length, and know **how many controller addresses** have some permissions on the UP.
```
Comment (fhildeb) [inline docs/guides/key-manager/get-controllers.md:51]:
```suggestion
1. query the `AddressPermissions[]` to get the array length and know **how many controller addresses** have some permissions on the UP.
```

File: docs/guides/key-manager/get-controllers.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+2. querying each index of the `AddressPermissions[index]` array to retrieve each address individually.
```
Comment (fhildeb) [inline docs/guides/key-manager/get-controllers.md:52]:
```suggestion
2. query each index of the `AddressPermissions[index]` array to retrieve each address individually.
```

File: docs/guides/key-manager/get-controllers.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+  myUniversalProfileAddress,
+  web3.currentProvider,
+);
+```
+
+## Step 2 - Get the list of controller addresses
+
+The next step is to retrieve the list of controller addresses. In other words, addresses that have some permissions set and are allowed to interact with the Universal Profile.
+
+All the controller addresses are listed under the data key `AddressPermissions[]`. We can therefore obtain this list of addresses by:
```
Comment (fhildeb) [inline docs/guides/key-manager/get-controllers.md:49]:
```suggestion
All the controller addresses are listed under the data key `AddressPermissions[]`. The list of addresses can be obtained in two steps:
```

Comment (fhildeb) [reply docs/guides/key-manager/get-controllers.md:49]: I would suggest keeping bulletpoints/orders in present.

File: docs/guides/key-manager/get-controllers.md [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+
+1. querying the `AddressPermissions[]` to get the array length, and know **how many controller addresses** have some permissions on the UP.
+2. querying each index of the `AddressPermissions[index]` array to retrieve each address individually.
+
+Thanks to _erc725.js_, we can do that very easily. The library will do both steps for us and will return the full list of controller addresses.
```
Comment (fhildeb) [inline docs/guides/key-manager/get-controllers.md:54]: 2x will in a row

```suggestion
Thanks to _erc725.js_, we can do that very easily. The library will do both steps for us and returns the full list of controller addresses.
```

File: docs/guides/key-manager/get-controllers.md [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+//   value: [
+//     '0x5F606b5b237623463a90F63230F9b929321dbCBa',
+//     '0xa1061408e55c971fD129eF5561dFB953d598dAD7'
+//   ]
+// }
+```
+
+## Step 3 - Get permissions of each controllers
+
+Getting the addresses of each controller is not enough. We need to retrieve the permissions of each of these controller addresses, so to know what they are allowed to do in our Universal Profile.
```
Comment (fhildeb) [inline docs/guides/key-manager/get-controllers.md:73]:
```suggestion
Getting the addresses of each controller is not enough. We need to retrieve the permissions of each controller address to know what they are allowed to do in our Universal Profile.
```

**Review by fhildeb — CHANGES_REQUESTED** [review]
Super cool- nice new picture, this way it can not be copy-pasted out of context. We should add headings to more pictures on the docs!

File: docs/guides/key-manager/get-controllers.md (hunk 3 above)
Comment (Hugoo) [reply docs/guides/key-manager/get-controllers.md:51]: I'd like to keep `querying` and remove the comma.

File: docs/guides/key-manager/get-controllers.md (hunk 4 above)
Comment (Hugoo) [reply docs/guides/key-manager/get-controllers.md:52]: I think `querying` is better than `query`

"this list of addresses by query the" doesn't sounds correct.

Comment (fhildeb) [reply docs/guides/key-manager/get-controllers.md:52]: true, would either reorg the sentence structure to have it in present, or leave present progressive then. whatever is more fluent

File: docs/guides/key-manager/get-controllers.md (hunk 5 above)
Comment (CJ42) [reply docs/guides/key-manager/get-controllers.md:49]: I think having _"listed"_ and then the word _"Iist of"_ makes it a bit repetitive in the end.
My suggestion:
```suggestion
All the controller addresses are listed under the data key `AddressPermissions[]`. We can retrieve them by:
```

**Review by fhildeb — APPROVED** [review]

File: docs/guides/key-manager/get-controllers.md (hunk 5 above)
Comment (fhildeb) [reply docs/guides/key-manager/get-controllers.md]: Good point, i like it.

**Review by CallumGrindle — APPROVED** [review]
Really cool guide

**Pull request merged**